### PR TITLE
[Refactoring] Replace id with instancetype in initializers

### PR DESCRIPTION
**Pull request by ejensen**

### CODE REVIEW

File: AsyncDisplayKit/ASImageNode.mm [hunk 1]
```diff
@@ -37,12 +37,12 @@ @interface _ASImageNodeDrawParameters : NSObject
...
 @implementation _ASImageNodeDrawParameters
 
-- (id)initWithImage:(UIImage *)image
-             bounds:(CGRect)bounds
-             opaque:(BOOL)opaque
-      contentsScale:(CGFloat)contentsScale
-    backgroundColor:(UIColor *)backgroundColor
-        contentMode:(UIViewContentMode)contentMode
+- (instancetype)initWithImage:(UIImage *)image
+                       bounds:(CGRect)bounds
```
Comment (nguyenhuy) [inline AsyncDisplayKit/ASImageNode.mm]: Tabs?

Comment (ejensen) [inline AsyncDisplayKit/ASImageNode.mm]: It indented using spaces. The change is due to the 1st line becoming longer. The subsequent lines are further indented in order to align `:`'s

Comment (nguyenhuy) [inline AsyncDisplayKit/ASImageNode.mm]: Got it. Sorry :)

**Comment by appleguy** [comment]
@ejensen  nifty, thanks!

**Pull request merged**